pkgs/development/haskell-modules/configuration-ghc-8.2.x.nix

```diff
@@ -89,7 +116,24 @@ self: super: {
   cabal2nix = super.cabal2nix.overrideScope (self: super: { Cabal = self.Cabal_2_2_0_1; });
   cabal2spec = super.cabal2spec.overrideScope (self: super: { Cabal = self.Cabal_2_2_0_1; });
   distribution-nixpkgs = super.distribution-nixpkgs.overrideScope (self: super: { Cabal = self.Cabal_2_2_0_1; });
-  hackage-db_2_0_1 = super.hackage-db_2_0_1.overrideScope (self: super: { Cabal = self.Cabal_2_2_0_1; });
+  hackage-db_2_0_1 = (self.callPackage
```

Comment:
Why did you add this Nix build into the overrides file?

Reply:
IDK what's the policy regarding keeping old version pinned derivations (and whether it's important to preserve them).

I assume that according to your comment about Cabal_1_x, no one cares about these, so I just removed them. Let me know if something other is desired.